stdlib/public/core/Integers.swift.gyb

```diff
@@ -2212,7 +2207,65 @@ ${operatorComment(x.nonMaskingOperator, True)}
 
 extension FixedWidthInteger {
   @_semantics("optimize.sil.specialize.generic.partial.never")
-  public init<Other: BinaryInteger>(clamping source: Other) {
+  public // @testable
+  static func _convert<Source : BinaryFloatingPoint>(
+    from source: Source
+  ) -> (value: Self?, exact: Bool) {
```

Comment:
I must be missing something. Is `(value: 42, exact: false)` a possible result? To me it looks like the nullability of `value` holds enough information.